### ISSUE TITLE
[UI] Some venues unable to open up Google Map

### ISSUE THREAD

**Issue by tlylt** [wilsonngja/MeetUps#31]
To reproduce:
![image](https://user-images.githubusercontent.com/41845017/176186575-c0455490-2ad0-4fdb-b939-88ab627c6889.png)
- Clicking on `LAW_CR3-7` or anything starting with `LAW` is not able to open up a Google Map

**Comment by wilsonngja** [comment]
![image](https://user-images.githubusercontent.com/31720838/176483657-9ed69ae4-76c6-4fce-b44b-d1f5dd246f3a.png)

Hi @tlylt, I just tried on my end and this is what it's shown. Can I check with you if it's the same?

**Comment by tlylt** [comment]
> ![image](https://user-images.githubusercontent.com/31720838/176483657-9ed69ae4-76c6-4fce-b44b-d1f5dd246f3a.png)
> 
> Hi @tlylt, I just tried on my end and this is what it's shown. Can I check with you if it's the same?

Yes, I think somehow when I tested it previously nothing happened. If this is the case, perhaps explain why unable to display the location in the prompt?

**Comment by tlylt** [comment]
> Hmm that's weird because what our code does is that we will look up the json file containing the latitude and longitude of the location (key).
> 
> If it doesn't exist then it will show the error message that you are seeing.

Sure if it is not reproducible perhaps there was an issue with my browser, can close this issue if no further action required :)
